### PR TITLE
packages apt: enable test during building deb package

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

File: packages/Rakefile [hunk 1]
```diff
@@ -63,10 +63,7 @@ class GroongaPackageTask < PackageTask
     end
     yum_spec_in_file = File.join(__dir__, "groonga.spec.in")
     original_spec_in_file = File.join(__dir__, "rpm", "centos", "groonga.spec.in")
-    file yum_spec_in_file => original_spec_in_file do |task|
-      cp(original_spec_in_file,
-         task.name)
-    end
+    cp(yum_spec_in_file, yum_spec_in_file)
```
Comment (kou) [inline packages/Rakefile]: You should not run any actions outside of task.
This runs with `rake -T`.

File: packages/apt/debian-buster/Dockerfile [hunk 2]
```diff
@@ -15,19 +15,42 @@ RUN \
...
+  apt install -y -V ${quiet} \
+    cutter-testing-framework && \
   apt clean && \
   rm -rf /var/lib/apt/lists/*
+
+RUN \
+  git clone --depth 1 https://github.com/sstephenson/rbenv.git ${HOME}/.rbenv && \
+  git clone --depth 1 https://github.com/sstephenson/ruby-build.git ${HOME}/.rbenv/plugins/ruby-build && \
+  CONFIGURE_OPTS="--disable-install-rdoc" ${HOME}/.rbenv/bin/rbenv install 2.6.5 && \
+  ${HOME}/.rbenv/bin/rbenv global 2.6.5
```
Comment (kou) [inline packages/apt/debian-buster/Dockerfile]: Why can not use system Ruby?

File: packages/debian/rules [hunk 3]
```diff
@@ -10,15 +10,18 @@ export DEB_BUILD_HARDENING = 1
...
 %:
 	dh $@
 
 override_dh_auto_configure:
-	dh_auto_configure -- --with-munin-plugins --enable-mruby
+	dh_auto_configure -- --with-munin-plugins --with-ruby=$(shell rbenv which ruby) --enable-mruby
 
-# disable 'make check'.
 override_dh_auto_test:
+	bash -x ./test/unit/run-test.sh
```
Comment (kou) [inline packages/debian/rules]: Is `bash -x` needed?

File: packages/yum/centos-6/Dockerfile [hunk 4]
```diff
@@ -5,28 +5,40 @@ ARG DEBUG
 
 RUN \
   quiet=$([ "${DEBUG}" = "yes" ] || echo "--quiet") && \
-  yum install -y https://packages.groonga.org/centos/groonga-release-latest.noarch.rpm && \
   yum update -y ${quiet} && \
   yum install -y ${quiet} epel-release centos-release-scl && \
+  yum install -y https://packages.groonga.org/centos/groonga-release-latest.noarch.rpm && \
+  yum install -y http://iij.dl.osdn.jp/storage/g/c/cu/cutter/centos/cutter-release-1.3.0-1.noarch.rpm && \
```
Comment (kou) [inline packages/yum/centos-6/Dockerfile]: Why is https://osdn.net/projects/cutter/storage/centos/cutter-release-1.3.0-1.noarch.rpm not used?

**Comment by kou** [comment]
I hope that you don't merge your pull request by yourself.
Normally, your pull request has several concerns without review.

File: packages/apt/debian-buster/Dockerfile (hunk 2 above)
Comment (kenhys) [reply packages/apt/debian-buster/Dockerfile]: grntest requires newer ruby. at least >= 2.2.

File: packages/yum/centos-6/Dockerfile (hunk 4 above)
Comment (kenhys) [reply packages/yum/centos-6/Dockerfile]: It should be , but It seems that sometimes fails to fetch from osdn.net.

File: packages/debian/rules (hunk 3 above)
Comment (kenhys) [reply packages/debian/rules]: Yes, It may be helpful when test has failed.

**Pull request merged**